tasks/github.py

```diff
@@ -59,3 +59,44 @@ def trigger_macos_test(
     follow_workflow_run(run_id)
 
     download_artifacts(run_id, destination)
+
+
+@task
+def lint_codeowner(ctx):
+    """
+    Check every package in `pkg` has an owner
+    """
+
+    base = os.path.dirname(os.path.abspath(__file__))
+    root_folder = os.path.join(base, "..")
+    os.chdir(root_folder)
+
+    owners = get_code_owners(root_folder)
+    
+    # make sure each root package has an owner        
+    pkgs_without_owner = find_packages_without_owner(owners, "pkg")        
+    if len(pkgs_without_owner) > 0:
+        raise Exit(f'The following root packages don\'t have an owner in CODEOWNERS: {pkgs_without_owner}')    
```

Comment:
nit
```suggestion
        raise Exit(f'The following packages  in `pkg` directory don\'t have an owner in CODEOWNERS: {pkgs_without_owner}', code=1)    
```

PS: I know `code` isn't strictly required but it's better to be explicit since the invoke API is a bit lacking here.